### PR TITLE
Added Indexer jobs

### PR DESCRIPTION
**Pull request by prabhat-webkul**
Added Indexer jobs

### CODE REVIEW

File: packages/Webkul/Product/src/Jobs/Indexers/ElasticSearch.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function handle()
+    {
+        $this->products->map(function ($product_id, $index) {
+            $this->products[$index] = app(
+                ProductRepository::class
+            )
+                ->findOrFail($product_id);
+
+            return;
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Jobs/Indexers/ElasticSearch.php:60]: No need for a return, if you are using `findOrFail`.

File: packages/Webkul/Product/src/Jobs/Indexers/Inventory.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function handle()
+    {
+        $this->products->map(function ($product_id, $index) {
+            $this->products[$index] = app(
+                ProductRepository::class
+            )
+                ->findOrFail($product_id);
+
+            return;
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Jobs/Indexers/Inventory.php:60]: Same here.

File: packages/Webkul/Product/src/Jobs/Indexers/Price.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+     */
+    public function handle()
+    {
+        $this->products->map(function ($product_id, $index) {
+            $this->products[$index] = app(
+                ProductRepository::class
+            )
+                ->findOrFail($product_id);
+
+            return;
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Jobs/Indexers/Price.php:60]: Same here.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Product/src/Jobs/Indexers/ElasticSearch.php (hunk 1 above)
Comment (prabhat-webkul) [reply packages/Webkul/Product/src/Jobs/Indexers/ElasticSearch.php]: added changes

File: packages/Webkul/Product/src/Jobs/Indexers/Inventory.php (hunk 2 above)
Comment (prabhat-webkul) [reply packages/Webkul/Product/src/Jobs/Indexers/Inventory.php]: added changes

File: packages/Webkul/Product/src/Jobs/Indexers/Price.php (hunk 3 above)
Comment (prabhat-webkul) [reply packages/Webkul/Product/src/Jobs/Indexers/Price.php]: added changes

**Pull request closed without merging**